src/App.tsx

```diff
@@ -1,14 +1,63 @@
-import React from 'react';
+import React, {
+  useCallback, useMemo, useRef, useState,
+} from 'react';
+
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { People } from './components/People';
+
+type Props = {
+  delay: number
+};
+
+export const App: React.FC<Props> = ({ delay }) => {
+  const [user, setUser] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [isFocused, setIsFocused] = useState(false);
+  const [selectedUser, setSelectedUser] = useState('');
+
+  const timerId = useRef(0);
+
+  const findUser = (username: string) => {
+    return peopleFromServer.find(person => person.name === username);
+  };
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
let's implement this requirement:
![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/265226ac-4877-4f9b-bc0e-a372a1a98cc3)